test/trento/domain/sap_system/sap_system_test.exs

```diff
@@ -323,13 +323,12 @@ defmodule Trento.SapSystemTest do
       )
     end
 
-    test "should register a SAP System and add an application instance" do
+    test "should register a SAP System and add an application instance when is already present a MESSAGESERVER instance and one ABAP instance is added" do
```

Comment:
`... when a MESSAGESERVER instance is already present and a new ABAP instance is added`

Reply:
done